src/UnityMvvmToolkit.UnityPackage/Assets/Plugins/UnityMvvmToolkit/Runtime/UITK/BindableUIElements/BindableDropdownField.cs

```diff
@@ -0,0 +1,180 @@
+
+        private PropertyBindingData _selectedItemBindingData;
+        private PropertyBindingData _itemsSourceBindingData;
+        
+        
+        public void Initialize()
+        {
+            choices = new List<string>();
+        }
+
+        public void Dispose()
+        {
+            choices.Clear();
+        }
+        
+        public void SetBindingContext(IBindingContext context, IObjectProvider objectProvider)
+        {
+            if (string.IsNullOrWhiteSpace(BindingItemsSourcePath) == false)
+            {
+                _itemsSourceBindingData ??= BindingItemsSourcePath.ToPropertyBindingData();
+                _itemsSource = objectProvider
+                    .RentReadOnlyProperty<ObservableCollection<string>>(context, _itemsSourceBindingData);
+                _itemsSource.Value.CollectionChanged += OnItemsCollectionChanged;
+                choices = new List<string>(_itemsSource.Value);
+            }
+            
+            if (string.IsNullOrWhiteSpace(BindingSelectedItemPath) == false)
+            {
+                _selectedItemBindingData ??= BindingSelectedItemPath.ToPropertyBindingData();
+                _selectedItemProperty = objectProvider.RentProperty<string>(context, _selectedItemBindingData);
+                _selectedItemProperty.ValueChanged += OnSelectedItemValueChanged;
+                UpdateControlValue(_selectedItemProperty.Value);
+                this.RegisterValueChangedCallback(OnControlValueChanged);
+                _selectedItemProperty.Value = choices[0];
+            }
+        }
+
+        private void OnItemsCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
+        {
+            switch (e.Action)
+            {
+                case NotifyCollectionChangedAction.Add:
+                    
+                    if (e.NewItems.Count != 1)
+                    {
+                        throw new NotSupportedException("RangeActionsNotSupported");
+                    }
+                    
+                    if (e.NewItems?[0] is string newValue)
+                    {
+                        choices.Add(newValue);
+                    }
+                    break;
+                
+                case NotifyCollectionChangedAction.Remove:
+
+                    if (e.OldItems.Count != 1)
+                    {
+                        throw new NotSupportedException("RangeActionsNotSupported");
+                    }
+
+                    if (e.OldStartingIndex < 0)
+                    {
+                        throw new InvalidOperationException("RemovedItemNotFound");
+                    }
+                    
+                    
+                    if (e.OldItems?[0] is string oldValue)
+                    {
+                        choices.Add(oldValue);
+                    }
+                    break;
+                
+                case NotifyCollectionChangedAction.Replace:
```

Comment:
I think we don't need to support `Replace`, `Move`, and `Reset` actions by default.
Look at the `OnItemsCollectionChanged` method implementation of the `BindableScrollView`.

Reply:
In this case, how will the user receive a notification about a change in the collection? For example, if we replaced the value in the List? I think it's worth keeping.

Reply:
How often does the Dropdown collection change? It is unlikely to change then change. But if the User wants to `Move`, `Replace`, and `Reset` actions support. It will be his job to implement them. Just make the `OnItemsCollectionChanged` method `protected` and `virtual`.

Reply:
@ChebanovDD  I fixed it